crates/sui-benchmark/tests/simtest.rs

```diff
@@ -261,6 +262,7 @@ mod test {
             transfer_object_weight,
             delegation_weight,
             batch_payment_weight,
+            adversarial_weight,
```

Comment:
this should be good enough - it might be worth double checking that you actually see the new workload in the logs, but this will run the new workload in CI and nightly simtests.

Reply:
Thanks! Is looking at the logs from the command in the test plan good enough, or is there a different command I should run (or CI job to look at)?

Reply:
Disregard--I see it in the PR CI!